### ISSUE TITLE
virtlet should create /var/lib/virtlet if it doesn't exist

### ISSUE THREAD

**Issue by pskrzyns** [Mirantis/virtlet#41]
```bash
./virtlet
libvirt: Storage Driver error : Storage pool not found: no storage pool with matching name 'default'
libvirt: Storage Driver error : Storage pool not found: no storage pool with matching name 'volumes'
libvirt: Storage Driver error : cannot open path '/var/lib/virtlet': No such file or directory
E0926 18:33:22.935635   14979 virtlet.go:45] Initializing server failed: &errors.errorString{s:"cannot open path '/var/lib/virtlet': No such file or directory"}
```

**Comment by jellonek** [comment]
Could you provide steps to reproduce this?
Using correct procedure to start (described in `README.md`) - `cd contrib/docker-compose; docker-compose up` this directory is created properly.

**Comment by nhlfr** [comment]
He just uses autotools in his host system and expects that virtlet will create this dir (which we create just by creating a docker volume).

We may fix that later, but TBH we probably wouldn't consider that as a very important issue.

**Comment by pskrzyns** [comment]
@jellonek the steps you described are in section running local environment  (https://github.com/Mirantis/virtlet#running-local-environment).
I just builde virtlet localy and then tried to run it on remote node of kubernetes cluster.

**Comment by jellonek** [comment]
Closed by #58
